staging/src/k8s.io/kubectl/pkg/kuberc/kuberc_test.go

```diff
@@ -936,18 +942,35 @@ func TestApplyOverride(t *testing.T) {
 			if errWriter.String() != "" {
 				t.Fatalf("unexpected error message %s\n", errWriter.String())
 			}
+			// Verify annotation and the original command
+			var originalCommand string
+			if originalCommand, ok = actualCmd.Annotations[KubeRCOriginalCommandAnnotation]; !ok {
+				t.Fatalf("unable to find the KubeRCOriginalCommandAnnotation")
+			}
 
+			// If an alias is detected, take its command
+			for _, alias := range testPreference.Aliases {
```

Comment:
As mentioned above, this should not be needed. You should work directly from the command, not the preference.